src/seedu/addressbook/parser/Parser.java

```diff
@@ -3,13 +3,14 @@
 import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Collections;
+import java.util.Collection;
+import java.util.Arrays;
+import seedu.addressbook.commands.ThenListCommand;
```

Comment:
should this be put on the following block instead? considering its importing from `seedu`

Reply:
Yes, that's right. Thanks for the notice.